app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,41 @@
+<div class="container">
+  <div class="row justify-content-center">
+    <div class="col-md-6">
+      <div class="card shadow">
+        <div class="card-body">
+
+<h2 class="text-center">Log in</h2>
+
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field mb-3">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", class: "form-control", id: "exampleInputEmail1", "aria-describedby" => "emailHelp" %>
+  </div>
+
+  <div class="field mb-3">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", class: "form-control", id: "exampleInputPassword1" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me, class: "form-check-input", id: "exampleCheck1" %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in", class: "btn btn-warning my-3" %>
+  </div>
+<% end %>
+
```

Comment:
- [ ] Great job on the project requirements but there's one style issue. Please make sure the login button has complete width and the "Log in" text is in the center to follow the wireframe design as requested in the project requirements.

### Your design

![image](https://github.com/vangelif/blog_app/assets/83803414/42bcc549-f2fe-4866-8f7d-2deee2d20152)

### Wireframe design

![image](https://github.com/vangelif/blog_app/assets/83803414/7388d970-5f23-498e-b419-e7be9287ef34)